client/src/components/Checkbox.js

```diff
@@ -0,0 +1,17 @@
+import { useState } from "react";
+import Checkbox from "../assets/icon/checkbox.svg";
+import CheckboxChecked from "../assets/icon/checkboxChecked.svg";
+
+export default function GetCheckBox() {
+  const [ischecked, setIsChecked] = useState(false);
```

Comment:
```suggestion
  const [isChecked, setIsChecked] = useState(false);
```

for better readability, I would use camelCase for "isChecked"

Reply:
Your absolutely right! Thx for noticing!